packages/shared/src/file.system.ts

```diff
@@ -96,9 +96,6 @@ s3Fs.credentials = credentials;
 
 fsa.register('https://', new FsHttp());
 fsa.register('s3://', s3Fs);
-fsa.register('s3://nz-imagery', s3FsPublic);
```

Comment:
Do we still need `s3FsPublic` as it is not been used anywhere.

Reply:
its exported so wasnt sure if any dependent packages were using it

Reply:
Cool, happy to keep it.